contracts/staker/curve/implementations/arbitrum/ConvexTokenStakerArbitrum.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.17;
+
+import "../../ConvexTokenStaker.sol";
+import { IConvexBaseRewardPoolSideChain, EarnedData } from "../../../../interfaces/external/convex/IBaseRewardPool.sol";
+
+/// @title ConvexTokenStakerArbitrum
+/// @author Angle Labs, Inc.
+/// @dev Constants for borrow staker adapted to Curve LP tokens deposited on Convex Arbitrum
+abstract contract ConvexTokenStakerArbitrum is ConvexTokenStaker {
+    /// @inheritdoc ERC20Upgradeable
```

Comment:
no `claimableRewards` for arbitrum?

Reply:
there is an implementation int `BorrowStaker`, it is only overidden when the function can be made `view`